front/lib/resources/group_resource.ts

```diff
@@ -420,11 +417,11 @@ export class GroupResource extends BaseResource<GroupModel> {
     const userIds = users.map((u) => u.sId);
     const userResources = await UserResource.fetchByIds(userIds);
     if (userResources.length !== userIds.length) {
-      return new Err(
-        userIds.length === 1
-          ? new Error("The user was not found.")
-          : new Error("Some users were not found.")
-      );
+      new Err({
```

Comment:
It's weird that this work, given that `GroupError` is defined as extending `Error` 🤔

Reply:
Seems this code snapshot was prior to converting to Error type .. it has been replaced by `new Err(new GroupError`

Reply:
Just checked out your branch and it's not a proper error as well 🤔

Reply:
found it .. seems the type were compatible , bit was not a proper Error indeed